### PR TITLE
Updating the amor documentation plotting to use plopp

### PR DESCRIPTION
**Pull request by arm61**
As in the title. Note that I have removed the instrument view. I plan to replace this with a plot of the logical x- and y-dimensions summed along z once https://github.com/scipp/ess/pull/139 is merged.

### CODE REVIEW

**Comment by arm61** [comment]
Okay I can't work out what I am doing wrong to add `plopp` to the repo...

**Comment by nvaytet** [comment]
> Okay I can't work out what I am doing wrong to add plopp to the repo...

I got a passing build by moving `plopp` into the conda deps, but as far as I can tell, it should have worked in `pip` as well. I think just re-running the workflow with your last commit would have worked,

In any case, putting `plopp` alongside `scipp`, and `scippneutron` makes more sense I think.

**Comment by arm61** [comment]
Awesome cheers! These isn’t a lot of changes in this commit but more to come hopefully.

File: docs/instruments/amor/amor_reduction.ipynb [hunk 1]
```diff
@@ -315,11 +299,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c7e4ecca-bb23-4b6c-a056-ebb9dcfa579d",
+   "id": "ba03ff0f-0377-4e4f-8a0f-3f4c987c002c",
    "metadata": {},
    "outputs": [],
    "source": [
-    "sample_wav.bins.concatenate('detector_id').plot()"
+    "sample_wav.bins.concat('detector_id').hist(wavelength=200).plot()"
```
Comment (nvaytet) [inline docs/instruments/amor/amor_reduction.ipynb:306]: One question: why are we now seeing errorbars while we didn't with the old plotting?
Screenshot at 2022-11-03 09-32-01

**Review by nvaytet — COMMENTED** [review]
Looks good overall, just one question.

File: docs/instruments/amor/amor_reduction.ipynb (hunk 1 above)
Comment (arm61) [reply docs/instruments/amor/amor_reduction.ipynb:306]: 🤷

Comment (arm61) [reply docs/instruments/amor/amor_reduction.ipynb:306]: Suggestions:
- It is a difference between `scipp` plots and `plopp`?
- It is a difference between `concatenate` and `concat`?

Comment (nvaytet) [reply docs/instruments/amor/amor_reduction.ipynb:306]: @SimonHeybrock any ideas?

Comment (SimonHeybrock) [reply docs/instruments/amor/amor_reduction.ipynb:306]: We think it was probably a bug in the old plotting: It simply did not show error bars when auto-histogramming data.

**Review by nvaytet — APPROVED** [review]

**Pull request merged**